grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -299,7 +299,7 @@ class Requisition implements Comparable<Requisition>, Serializable {
                 requestedByName: requestedBy?.name,
                 description: description,
                 dateRequested: dateRequested.format("MM/dd/yyyy"),
```

Comment:
Should `dateRequested` also be formatted?

Reply:
NOTE: This is a question, not a request. I'm asking because I'm not sure if it should or should not be a date time.

Reply:
Hmm, I am not sure if this is needed with time value (if this is the date that we are choosing on the first step). I believe that if we'd like to have a time there too it would be already requested by Jesse or Kelsey.

Reply:
@awalkowiak Ok sounds good. The only reason I brought it up is because the PR looked like it was dealing with all of the dates. But I hadn't seen Kelsey's comment in the ticket that basically scoped this to just the two dates. One thing to look into ... is dateRequested used to derive a value for either of those other two dates?